src/Conf.ml

```diff
@@ -439,6 +440,23 @@ module Formatting = struct
     C.choice ~names ~all ~doc ~section (fun conf x ->
         {conf with break_infix= x} )
 
+  let break_separators =
+    let doc =
+      "Indicates how are broken an expression with a separator when it \
+       does not fit on a single line."
```

Comment:
Perhaps reword: "Break before or after separators such as `;` in list or record expressions."